src/server/scripts/Northrend/Ulduar/Ulduar/ulduar.h

```diff
@@ -444,6 +450,24 @@ AI* GetUlduarAI(T* obj)
     return GetInstanceAI<AI, T>(obj, UlduarScriptName);
 }
 
+class KeeperDespawnEvent : public BasicEvent
```

Comment:
Sorry i think the later half of the changes on ulduar.h came in when they weren't part of this PR. Ill revert them